daemon/run_openrazer_daemon.py

```diff
@@ -39,6 +39,7 @@ def parse_args():
 
     parser.add_argument('-r', '--respawn', action='store_true', help='Stop any existing daemon first, if one is running.')
     parser.add_argument('-s', '--stop', action='store_true', help='Gracefully stop the existing daemon.')
+    parser.add_argument('--use-system-bus', action='store_true', help='Run on the system bus', default=False)
```

Comment:
I wonder if this is a good idea: I am trying to rebase my changes on top of yours, and I am now running into the issue that we do not know where the bus will appear.

I guess we can check one and the other if the first fails, but it'll be kind of difficult to know for sure why we do not connect if both fail. Wouldn't a complete switch be better?

Reply:
it breaks current clients and requires installation of the policy file. Long-term it *has* to change to the system bus anyway, IMO, so it's mostly a matter of whether there should be a transition period